ai2thor_wrapper/envs.py

```diff
@@ -0,0 +1,195 @@
+import os
+import json
+import configparser
+import sys
+
+import numpy as np
+import skimage.color, skimage.transform
+import ai2thor.controller
+
+POSSIBLE_ACTIONS = [
+    'MoveAhead',
+    'MoveBack',
+    'MoveRight',
+    'MoveLeft',
+    'LookUp',
+    'LookDown',
+    'RotateRight',
+    'RotateLeft',
+    'OpenObject',
+    'CloseObject',
+    'PickupObject',
+    'PutObject'
+    # Teleport and TeleportFull but these shouldn't be actions for an agent
+]
+
+class ThorWrapperEnv():
+    def __init__(self, scene_id='FloorPlan28', seed=None, task=0, max_episode_length=1000, current_object_type='Mug',
+                 grayscale=True, interaction=True, config_path='config_example.ini', movement_reward=0):
+        # Loads config file from path relative to ai2thor_wrapper python package folder
+        config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), config_path)
+        self.config = configparser.ConfigParser()
+        config_output = self.config.read(config_path)
+        if len(config_output) == 0:
+            print('No config file found at: {}. Exiting'.format(config_path))
```

Comment:
No print and exit. Raise it!
```suggestion
           raise ValueError('No config file found at: {}. Exiting'.format(config_path))
```

Reply:
yep, i knew you'd say that haha